### PR TITLE
[BUGFIX][MER-2434] Only return pages for upcoming activities

### PR DESCRIPTION
**Pull request by augusto1024**
[Link to the ticket](https://eliterate.atlassian.net/browse/MER-2434)

### CODE REVIEW

File: test/oli/sections_test.exs [hunk 1]
```diff
@@ -1530,19 +1536,76 @@ defmodule Oli.SectionsTest do
...
+        is_liveview: false
+      }
+
+      enroll_user_to_section(student, section, :context_learner)
+
+      next_activities =
+        Sections.get_next_activities_for_student(section.slug, student.id, session_context)
+
+      assert length(next_activities) == 1
+      assert Enum.at(next_activities, 0).title == "Upcoming assessment"
```
Comment (simonchoxx) [inline test/oli/sections_test.exs:1608]: I like the use of `Enum.at` here (I usually use List.first)!!!

**Review by simonchoxx — APPROVED** [review]
Great work 🚀🚀 🚀

I would only add that there are some lines that were modified (maybe because of the formatting) and I understand that they do not correspond to the changes needed for the ticket!

File: test/oli/sections_test.exs (hunk 1 above)
Comment (augusto1024) [reply test/oli/sections_test.exs:1608]: I think I actually like `List.first` better 😅. I don't know why I used `Enum.at`

**Comment by augusto1024** [comment]
@simonchoxx yeah, everytime I make changes in a file, I run the formatter for it. We should open a PR for running it across the whole project

File: test/oli/sections_test.exs (hunk 1 above)
Comment (simonchoxx) [reply test/oli/sections_test.exs:1608]: We get the same thing with both functions, but I was glad to see it because I don't think I've used it!

**Review by darrensiegel — APPROVED** [review]

**Pull request merged**